src/SignalR/common/SignalR.Common/test/Internal/Protocol/MessagePackHubProtocolTestBase.cs

```diff
@@ -0,0 +1,434 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
```

Comment:
This makes me a bit sad.

Add a comment explaining that this is shared with Razor Components and when adding tests put it in MessagePackHubProtocolTests.cs unless they are simple tests.